package.json

```diff
@@ -38,6 +38,11 @@
         "icon": "resources/dark/add-location.svg",
         "category": "Servers"
       },
+      {
```

Comment:
also add it to "view/item/context" array below with condition to enable for started server.